ome_zarr/io.py

```diff
@@ -53,165 +100,101 @@ def exists(self) -> bool:
         """Return true if either zgroup or zarray metadata exists."""
         return self.__exists
 
+    @property
+    def store(self) -> FSStore:
+        """Return the initialized store for this location"""
+        assert self.__store is not None
+        return self.__store
+
     @property
     def root_attrs(self) -> JSONDict:
         """Return the contents of the zattrs file."""
         return dict(self.__metadata)
 
-    def load(self, subpath: str = "") -> da.core.Array:
+    def load(self, subpath: str = "", nested: bool = False) -> da.core.Array:
```

Comment:
Going forward, we expect all future specs will be nested, so would it make sense for the default `nested: bool = True`?